### PR TITLE
Add slack and stackoverflow icon to footer in docs

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: docs/source/templates/layout.html [hunk 1]
```diff
@@ -35,6 +35,12 @@
       <a href="https://twitter.com/AlteryxOSS" target="_blank">
         <img  class="footer-image-twitter" src="{{ pathto('_static/images/twitter.svg', 1) }}" alt="Twitter">
       </a>
+      <a href="https://join.slack.com/t/alteryx-oss/shared_invite/zt-6inxevps-RSbpr9lsACE1kObXz4rIuA" target="_blank">
```
Comment (thehomebrewnerd) [inline docs/source/templates/layout.html:38]: This Slack link is not active

Comment (gsheni) [reply docs/source/templates/layout.html]: Woops, out `stable` branch has the wrong link

Comment (gsheni) [reply docs/source/templates/layout.html]: Fixed:
https://github.com/alteryx/featuretools/pull/2087/commits/fd41e49333971b14655ef85305787f004aceeb0b

**Review by thehomebrewnerd — APPROVED** [review]
LGTM!

**Pull request merged**